src/main/java/com/botdetector/ui/BotDetectorPanel.java

```diff
@@ -917,11 +944,13 @@ public void setPrediction(Prediction pred, PlayerSighting sighting)
 				CaseInsensitiveString name = normalizeAndWrapPlayerName(pred.getPlayerName());
 
 				// If the player has already been feedbacked/flagged, ensure the panels reflect this
-				resetFeedbackPanel();
+				resetFeedbackPanel(true);
 				Boolean feedbacked = plugin.getFeedbackedPlayers().get(name);
 				if (feedbacked != null)
 				{
 					disableAndSetColorOnFeedbackPanel(feedbacked);
+					String txt = plugin.getFeedbackedPlayersText().get(name);
+					feedbackTextbox.setText(txt != null ? txt : "");
```

Comment:
Actually I might want to have this if statement outside, for when sending a feedback fails and we want to restore whatever was last tried to be sent.